mora/service/employee.py

```diff
@@ -36,6 +37,13 @@
                             url_prefix='/service')
 
 
+RELATION_TYPES = {
+    'it': itsystem.ITSystems,
+    'address': address.Addresses,
+    'org_unit': orgunit.OrgUnit,
```

Comment:
This allows us to now potentially create and edit org units through the employee endpoints. Does that make sense?